src/python/turicreate/data_structures/sframe_builder.py

```diff
@@ -58,7 +58,7 @@ class SFrameBuilder(object):
 
     >>> sb.append_multiple([[2,2.0,"2"],[3,3.0,"3"]])
 
-    >>> sb.close()
+    >>> new_sf = sb.close()
```

Comment:
The output shown here doesn't match what the code will produce. I like the idea of using a temporary variable. I would add `print(new_sf)` so the code will produce the output shown.

Reply:
yeah! That's a good idea! Thanks!